bin/delete-marked-accounts

```diff
@@ -8,6 +8,8 @@ Includes:
 * Account liveness/status data (in Redis).
 
 """
+from __future__ import absolute_import
```

Comment:
This is needed because on Python 2 inbox.logging would shadow stdlib logging unless absolute_import is turned on (Python 3 behavior)